src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -183,9 +184,11 @@ class CodyToolWindowContent(private val project: Project) {
         project: Project,
         myAction: CodyToolWindowContent.() -> Unit
     ) {
-      if (!project.isDisposed) {
-        val codyToolWindowContent = project.getService(CodyToolWindowContent::class.java)
-        codyToolWindowContent.myAction()
+      UIUtil.invokeLaterIfNeeded {
```

Comment:
FYI i dont know if its a good idea to put this method everywhere, this is from documentation:

> Please use Application.invokeLater() with a modality state (or ModalityUiUtil, or TransactionGuard methods), unless you work with Swings internals and 'runnable' deals with Swings components only and doesn't access any PSI, VirtualFiles, project/module model or other project settings. For those, use ModalityUiUtil, application.invoke* or TransactionGuard methods.
On AWT thread, invoked runnable immediately, otherwise do SwingUtilities.invokeLater(Runnable) on it.